### ISSUE TITLE
Rollershutter: Enable manual positioning / Basic UI UX

### ISSUE THREAD

**Issue by cribskip** [openhab/openhab-android#679]
Basic UI enables the user to keep the UP/DOWN buttons pressed and releasing them STOPs the shutter.

IMO this feature should be also available in the app as its simpler to just keep the button pressed than pressing stop at that exact moment.

**Comment by dsacchet** [comment]
Indeed, it should be great if we can have a mix between Dimmer and Rollershutter BasicUI widget.
Something like : 
50% [-] -----X----- [+] [X]

**Comment by cribskip** [comment]
@mueller-ma thank you very much for the implementation, it's working just fine! 👍
